tests/internals.rs

```diff
@@ -195,7 +243,7 @@ fn declare_tx() -> InternalDeclare {
 #[test]
 fn test_declare_tx() {
     let (general_config, mut state) = create_account_tx_test_state().unwrap();
-
+    assert_eq!(&state, &expected_state_before_tx());
```

Comment:
This works no problem, but since you have both sides available by value you don't really need `&_` before them.
```suggestion
    assert_eq!(state, expected_state_before_tx());
```